Tools/autotest/vehicle_test_suite.py

```diff
@@ -4460,7 +4460,7 @@ def TestLogDownloadMAVProxyNetwork(self, upload_logs=False):
             self.mavproxy_load_module(mavproxy, 'log')
             self.wait_heartbeat()
             mavproxy.send("log list\n")
-            mavproxy.expect("numLogs")
+            mavproxy.expect("numLogs", timeout=120)
```

Comment:
This is probably not going to work.

UDPMulticast and UDPBroadcast are lossy.  Neither MAVProxy nor the autotest suite have retry logic to account for that.

I added a few notes over here (which should probably be on Buzz's issue): https://github.com/ArduPilot/ardupilot/pull/26371

Reply:
Oh i see. fair enough. I'll close this then. Why does it look like it is only happening with Sub, though?

Reply:
oh, I just remembered why I made this:
https://github.com/ArduPilot/ardupilot/actions/runs/8178212379/job/22362301977#step:7:19363

it does look like a log list comes through after the timeout.

Reply:
I guess there's an extra message from mavproxy  saying the total number of logs? that would make sense.

Reply:
> Oh i see. fair enough. I'll close this then. Why does it look like it is only happening with Sub, though?

The sub test suite is where the new tests for log-download-over multicast were put....